docs/basics/101-130-yodaproject.rst

```diff
@@ -0,0 +1,600 @@
+option is specified, siblings will be created *recursively*, i.e., not only a
+repository for the ``midterm_project`` dataset will be created, but also one
+for the ``iris_data`` dataset:
+
+
+.. ifconfig:: internal
+
+    .. runrecord:: _examples/DL-101-130-116
+       :language: console
+       :workdir: dl-101/DataLad-101/midterm_project
+       :realcommand: datalad --log-level critical siblings add -d . --name github --url https://github.com/adswa/midtermproject.git
+
+.. code-block:: bash
+
+   $ datalad create-sibling-github -d . -r midtermproject
+   .: github(-) [https://github.com/adswa/midtermproject.git (git)]
+   .: github(-) [https://github.com/adswa/midtermproject-input.git (git)]
+   'https://github.com/adswa/midtermproject.git' configured as sibling 'github' for <Dataset path=/home/me/dl-101/DataLad-101/midterm_project>
+   'https://github.com/adswa/midtermproject-input.git' configured as sibling 'github' for <Dataset path=/home/me/dl-101/DataLad-101/midterm_project/input>
+
+
+Verify that this worked by listing the siblings of the dataset:
+
+.. runrecord:: _examples/DL-101-130-117
+   :language: console
+   :workdir: dl-101/DataLad-101/midterm_project
+
+   $ datalad siblings
+
+.. gitusernote::
+
+   Creating a sibling on GitHub will create a new empty repository under the
+   account that you provide and set up a *remote* to this repository. Upon a
+   :command:`datalad publish` to this sibling, your datasets history
+   will be pushed there.
+
+   .. index:: datalad command; publish
+
+On GitHub, you will see two new, empty repository with the names
+``midtermproject``, and ``inputs``. However, none of these repositories yet contain
+any of your dataset's history or files. This requires *publishing* the current
+state of the dataset to this sibling with the :command:`datalad publish`
+(:manpage:`datalad-publish` manual) command. As before, we do this recursively with
+``-r/--recursive``. Because this will publish the contents of *two* datasets,
+``midterm_project`` and ``inputs``, you will be asked twice to authenticate.
+
+.. code-block:: bash
+
+   $ datalad publish -r --to github
+   [INFO   ] Publishing <Dataset path=/home/me/dl-101/DataLad-101/midterm_project/input> to github
+   publish(ok): input (dataset) [pushed to github: ['[new branch]', '[new branch]']]
+   [INFO   ] Publishing <Dataset path=/home/me/dl-101/DataLad-101/midterm_project> to github
+   publish(ok): . (dataset) [pushed to github: ['[new branch]', '[new branch]']]
+   action summary:
+     publish (ok: 2)
+
+.. gitusernote::
+
+   The :command:`datalad publish` uses ``git push``, and ``git annex copy`` under
+   the hood. Publication targets need to either be configured remote Git repositories,
+   or Git-annex special remotes (if they support data upload).
+
+Here is one important detail, though: By default, your tags will not be published.
+This requires an additional :command:`git push` with the ``--tags`` option to the
```

Comment:
There is a reason we that is the case. Tags are viral. Once pushed the propagate into any clone. However, especially the way a tag is used here, one may not actually want that. Let's say I remove the tag locally. Later on I add a clone of a friend as a remote in order to fetch a contribution. They still have the old tag, and BANG I remove my removed tag back where I no longer wanted it.

Maybe just skip the tag-push part here.